podpac/core/node.py

```diff
@@ -353,6 +353,37 @@ def find_coordinates(self):
 
         raise NotImplementedError
 
+    def get_bounds(self, crs="default"):
+        """Get the full available coordinate bounds for the Node.
+
+        Arguments
+        ---------
+        crs : str
+            Desired CRS for the bounds.
+            If not specified, the default CRS in the podpac settings is used. Optional.
+
+        Returns
+        -------
+        bounds : dict
+            Bounds for each dimension. Keys are dimension names and values are tuples (hi, lo).
```

Comment:
@jmilloy It looks like you're actually returning (lo, hi), not (hi, lo) as the comment mentions. Also above.